center/Dockerfile

```
@@ -51,6 +53,9 @@ COPY ${STARTUP_FOLDER_PATH}/nginx.conf /etc/nginx/
 COPY ${STARTUP_FOLDER_PATH}/restartMonitorService.sh restartMonitorService.sh
 COPY ${STARTUP_FOLDER_PATH}/splitLog.sh splitLog.sh
 COPY ${STARTUP_FOLDER_PATH}/start.sh start.sh
+COPY uber_image/${CENTER_INIT_SQLITE} /hydra/data/${CENTER_INIT_SQLITE}
+COPY uber_image/record_release.apk /storage/local/pkgstore/2023/04/23/record_release.apk
```

```suggestion
COPY uber_image/hydra_lab_android_client_release.apk /storage/local/pkgstore/2023/04/23/hydra_lab_android_client_release.apk
```
I suggest we use new name for this apk. Also apply to the test app.

---

Update the dest file name. Source file would be copied from Common module.